### ISSUE TITLE
tensorflow version 2.0 not compatible

### ISSUE THREAD

**Issue by jerabaul29** [tensorforce/tensorforce#623]
@AlexKuhnle it seems that version 2.0 of tensorflow is not compatible with tensorforce, just for the records (know that it is mentioned in the readme that it is built agains 1.3).

We were working with @renfenghk on a project and getting weird results with 2.0. I think he downgraded now, so not sure if the errors are still available (@renfenghk do you still have the error messages?) (I cannot run it myself for now).

I think if you have time maybe migrating could be good some day :) Of course no hurry.

**Comment by AlexKuhnle** [comment]
Yep, it's not compatible. I want to try again at some point how difficult it would be if one just relies on using the 1.X-compatibility functionality in 2.0. Properly migrating to 2.0 is likely a big and non-trivial project, so won't happen soon.

**Comment by jerabaul29** [comment]
Ok, thanks for the confirmation. May be good to indicate explicitly in the readme that it is not compatible with 2.0.

Also when you move, could you consider using keras definition for the models as we had discussed a long time ago? :)

**Comment by AlexKuhnle** [comment]
[Keras layers](https://github.com/tensorforce/tensorforce/blob/master/tensorforce/core/layers/keras.py) should be supported, but I think you mean to use Keras model instead of Module (or in addition),right? I'm not sure whether that will be super easy, depending on what assumptions they make. What would be the advantage?

**Comment by AlexKuhnle** [comment]
You're right, of course, ideally this should all be translated to proper TF 2.0 practice.

**Comment by AlexKuhnle** [comment]
The latest version should now work with (only) TF 2.0, although in v1 compatibility in some cases.
